tor/core/user_interaction.py

```diff
@@ -71,8 +72,10 @@ def process_coc(post, config):
     # instead. If they're actually new, then send a message to slack.
     if result == 1:
         send_to_modchat(
-            f'<http://www.reddit.com/u/{post.author.name}|u/{post.author.name}>'
-            f' has just accepted the CoC! {modchat_emote}',
+            f'<{reddit_url.format('/user/' + post.author.name)}|u/{post.author.name}>'
```

Comment:
While it looks correct according to GitHub's syntax highlighting, this also doesn't work:

```
>>> def foo(bar): return bar
... 
>>> f'{foo('bar')}'
  File "<stdin>", line 1
    f'{foo('bar')}'
              ^                                                                                                                                                                                              
SyntaxError: invalid syntax
```

I suggest using double quotes, ie. `reddit_url.format("/user/" + post.author.name)`.

Reply:
oops part of that is on me. I didn't correct for that. Sorry!